src/objCalculator.js

```diff
@@ -19,6 +19,23 @@
   calculator.div(3, 2) // Retorno: 1;
 */
 
-const calculator = {};
+const calculator = {
+  add: (number1, number2) => {
+    let result = Math.floor(number1 + number2);
+    return result;
+  },
```

Comment:
```suggestion
  add: (number1, number2) => number1 + number2,
```
Se você coloca o Math.floor(), quando tentar somar 3.2 + 3.2 seu resultado vai ser 6.0 e não 6.4 que seria o correto, a mesma lógica vai para a multiplicação e subtração, apenas na divisão teria que deixar porquê é um requisito do projeto.